test/e2e/e2e-cli.sh

```diff
@@ -1,21 +1,27 @@
 #!/bin/sh
-
-npm run build
+set -e
 
 WORK_DIR="./local/cli"
 CONFIG_FILE="./config-cli-e2e.json"
 
+npm ci #Install all dependencies to allow for building
+npm run build
+npm ci --omit=dev #Remove dev dependencies
```

Comment:
I doubt it would make much difference time-wise but I think you _could_ get away with removing this step as the tarball produced by `npm pack` wont include `node_modules` so pruning them isn't necessary